PCbuild/fix_lineendings.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+#
+# Fixes line endings of batch files to ensure they remain LF.
```

Comment:
Well, they weren't CRLF, because Mercurial (I'm using hggit on this machine) fixed them all and they all got changed.

I can fix them back, but then the review is going to be unreadable, so check it all out before I do it :)